static/swagger-specs/unified-tags.yaml

```diff
@@ -0,0 +1,983 @@
+      required: true
+      schema:
+        type: string
+    x-api-key:
+      name: x-api-key
+      in: header
+      description: The API key which can be copied from your Experience Platform integration. For more information on how to obtain this value, visit the [authentication tutorial](http://www.adobe.com/go/platform-api-authentication-en).
+      required: true
+      schema:
+        type: string
+    x-sandbox-name:
+      name: x-sandbox-name
+      in: header
+      description: All resources in Experience Platform are isolated to specific virtual sandboxes. Requests to Platform APIs require a header that specifies the name of the sandbox in which the operation will take place. See the [sandboxes overview](https://adobe.com/go/sandbox-overview-en) for more information.
+      required: true
+      schema:
+        type: string
+    content-type:
+      name: Content-Type
+      in: header
+      description: Type of content being sent in the body of the request. Should be `application/json`.
+      required: true
+      schema:
+        type: string
+    start:
+      name: start
+      in: query
+      description: The location where the list of results starts from. You can use this to indicate the starting index for pagination of results.
+      schema: 
+        type: string
+    limitTagCategories:
+      name: limit
+      in: query
+      description: The maximum number of tag categories you want retrieved per page.
+      schema: 
+        type: string
+    propertyTagCategories:
+      name: property
+      in: query
+      description: The attribute you want to filter by when retrieving tag categories.
+      schema:
+        type: string
+    sortByTagCategories:
+      name: sortBy
+      in: query
+      description: The order in which the tag categories are sorted by. Supported values include `name`, `createdAt`, and `modifiedAt`.
+      schema:
+        type: string
+    sortOrderTagCategories:
+      name: sortOrder
+      in: query
+      description: The direction in which the tag categories are sorted by. Supported values include `asc` and `desc`.
+      schema:
+        type: string
+    tag-category-id:
+      name: TAG_CATEGORY_ID
+      in: path
+      description: The ID of the tag category you're retrieving.
+      required: true
+      schema:
+        type: string
+    limitTags:
+      name: limit
+      in: query
+      description: The maximum number of tags you want retrieved per page.
+      schema:
+        type: string
+    propertyTags:
+      name: property
+      in: query
+      description: The attribute you want to filter by when retrieving tags. Supported values include `name`, `archived`, and `tagCategoryId`.
+      schema:
+        type: string
+    sortByTags:
+      name: sortBy
+      in: query
+      description: The order in which the tags are sorted by. Supported values include `name`, `createdAt`, and `modifiedAt`.
```

Comment:
This just reads a little strangely to me. Perhaps:
"The attribute used to sort the order of the tags. Supported values include `name`, `createdAt`, and `modifiedAt`."